apps/allocations/app/components/App/Payouts.js

```diff
@@ -163,7 +168,7 @@ const PayoutsWrap = styled.div`
   ${breakpoint(
     'small',
     `
-    padding: 0 2rem 2rem 2rem;
+    padding: 0 0 2rem 0;
```

Comment:
In the description, you said:

> Styling should have stayed exactly the same...

but this looks like a change. What prompted it?

Reply:
This was a fix.. although every fix is a change, right? So this one was to make Payout table to be aligned to Accounts, as per:
![Screenshot from 2019-07-29 13-05-19](https://user-images.githubusercontent.com/34452131/62049486-a2f92000-b20f-11e9-9d85-2bb5a676e9cf.png)
It's not perfectly aligned on the design, but then again what we have does not perfectly reflect the design - still, 0 looks much better than 2em:
![Screenshot from 2019-07-29 14-48-28](https://user-images.githubusercontent.com/34452131/62049680-097e3e00-b210-11e9-8628-7f2e59d89d82.png)